src/block-management/inline-toolbar/index.js

```diff
@@ -52,18 +53,18 @@ export default class InlineToolbar extends React.Component<PropsType> {
 	}
 
 	render() {
+		const { testID } = this.props;
 		return (
 			<View style={ styles.toolbar } >
 				<ToolbarButton
-					accessibilityLabel={ __( 'Move up' ) }
-					label={ __( 'Move up' ) }
+					title={ __( `Move ${ testID } up` ) }
```

Comment:
Hey @JavonDavis !
These titles will be spoken out loud by VoiceOver. Probably is not the best to have a `testID` here 😞

Reply:
> though it's still not clear enough for a translator without a hint.

That's why I suggested including the block name as the variable in there but I also thought that we translated these. But we don't https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/block-management/block-picker.js#L70 ... Unless we get it translated here https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/block-management/block-picker.js#L34... 

Hmm does something more like `Paragraph block 1. Move up` or `Paragraph block 1. Move up to position 0` In this case it's two sentences and I'm not sure if I got this wrong but maybe this will be easier to translate? I think it's good for both VoiceOver and the tests if the name of the block is indicated in this identifier right? A user can be sure what block is there and being moved up or down or removed. This is just my thinking though, I'm not sure and we don't want to include anything that complicates the VoiceOver experience.

Reply:
Getting good labels for VoiceOver is not easy 😅
The best is to experience it by ourselves to have a general idea of what experience a VoiceOver user might have (including testing using VoiceOver blindfolded).

One important thing is to have the most meaningful part that describes the element at the beginning, so the user can move to the next one fast if that is not the element they are looking for.

So far I'm adding the text `Paragraph block" to describe the block itself, so if we start with the same string the Up button, the user will think that it is a new paragraph block already (unless they wait long enough to notice the difference).

 `Move up to row 0` sounds like the best to me so far. The current index is not really needed for VoiceOver and it would be an unnecessary extra load. Bit it seems to be necessary for UITest, right?
How does it sound something like: `Move up from row 2 to row 1`?

Reply:
It's not that the current index is needed because that can still be retrieved by looking after the from and before the to. I was just trying to make it easier to write selectors for the buttons but your point here, 

> One important thing is to have the most meaningful part that describes the element at the beginning, so the user can move to the next one fast if that is not the element they are looking for.

makes sense. There'll just be a more complicated selector here for finding the buttons, basically looking at the child and ensuring the numbers match, but definitely doable.

Reply:
Thank you for the effort!

Reply:
> What do you think? Maybe @rachelmcr could help us too :)

Sorry, I missed this ping earlier! I just want to note that for translation it's always better to use variables within a string (with a comment to the translators to explain what the variable contains), rather than string concatenation, so that translators can easily move a variable within the string to account for word order differences between languages.

Reply:
One bad thing is that here we can't write a comment for the string, so it gets a little bit more complicated.

Reply:
> One bad thing is that here we can't write a comment for the string, so it gets a little bit more complicated.

@etoledom I'm not sure I understand what you mean here

Reply:
On iOS we can add a comment that is send together with the string to translate to the translators. As far as I know, we can't send a comment like that from string created in the JavaScript side (unless we can and I didn't know?).

Reply:
Ohh I didn't know that actually. You might be right, I'm not very familiar with how that works right now.